arangod/Metrics/MetricsFeature.cpp

```diff
@@ -0,0 +1,524 @@
+    nameVersionTable.try_emplace(
+        "arangodb_http_request_statistics_http_delete_requests_total",
+        "arangodb_http_request_statistics_http_delete_requests");
+    nameVersionTable.try_emplace(
+        "arangodb_http_request_statistics_http_get_requests_total",
+        "arangodb_http_request_statistics_http_get_requests");
+    nameVersionTable.try_emplace(
+        "arangodb_http_request_statistics_http_head_requests_total",
+        "arangodb_http_request_statistics_http_head_requests");
+    nameVersionTable.try_emplace(
+        "arangodb_http_request_statistics_http_options_requests_total",
+        "arangodb_http_request_statistics_http_options_requests");
+    nameVersionTable.try_emplace(
+        "arangodb_http_request_statistics_http_patch_requests_total",
+        "arangodb_http_request_statistics_http_patch_requests");
+    nameVersionTable.try_emplace(
+        "arangodb_http_request_statistics_http_post_requests_total",
+        "arangodb_http_request_statistics_http_post_requests");
+    nameVersionTable.try_emplace(
+        "arangodb_http_request_statistics_http_put_requests_total",
+        "arangodb_http_request_statistics_http_put_requests");
+    nameVersionTable.try_emplace(
+        "arangodb_http_request_statistics_other_http_requests_total",
+        "arangodb_http_request_statistics_other_http_requests");
+    nameVersionTable.try_emplace(
+        "arangodb_server_statistics_server_uptime_total",
+        "arangodb_server_statistics_server_uptime");
+    // And one for rocksdb:
+    nameVersionTable.try_emplace("rocksdb_engine_throttle_bps",
+                                 "rocksdbengine_throttle_bps");
+
+    v2suppressions = {
+        // Note that if we ever need to suppress a metric which is
+        // coming from the statistics feature further code is needed
+        // there. This list is only considered for the normally registered
+        // metrics in the MetricsFeature.
+        "arangodb_maintenance_phase1_accum_runtime_msec_total",
+        "arangodb_maintenance_phase2_accum_runtime_msec_total",
+        "arangodb_maintenance_agency_sync_accum_runtime_msec_total",
+        "arangodb_maintenance_action_accum_runtime_msec_total",
+        "arangodb_maintenance_action_accum_queue_time_msec_total",
+        "arangodb_agency_supervision_accum_runtime_msec_total",
+        "arangodb_agency_supervision_accum_runtime_wait_for_replication_msec_"
+        "total",
+        "arangodb_load_current_accum_runtime_msec_total",
+        "arangodb_load_plan_accum_runtime_msec_total",
+        "arangodb_aql_slow_query_total",
+        "arangodb_scheduler_jobs_dequeued",
+        "arangodb_scheduler_jobs_submitted",
+        "arangodb_scheduler_jobs_done",
+    };
+
+    v1suppressions = {
+        "arangodb_scheduler_jobs_dequeued_total",
+        "arangodb_scheduler_jobs_submitted_total",
+        "arangodb_scheduler_jobs_done_total",
+    };
+  } catch (std::exception const& e) {
```

Comment:
Still thinking the whole try..catch block is unnecessary here.
If any of the `emplace` operations fails, startup will be aborted properly.

Reply:
I'm ready to change that, but in the next PR
https://github.com/arangodb/arangodb/pull/15249#discussion_r761027800

Reply:
Ok!